src/import/sst_service.rs

```diff
@@ -618,6 +623,74 @@ impl<E: Engine> ImportSstService<E> {
     }
 }
 
+fn check_local_region_stale(
+    region_id: u64,
+    epoch: &RegionEpoch,
+    region_info_accessor: Arc<dyn RegionInfoProvider>,
+) -> Result<()> {
+    let (tx2, rx2) = crossbeam::channel::bounded(1);
+    region_info_accessor
+        .find_region_by_id(
+            region_id,
+            Box::new(move |item| {
+                tx2.send(item).unwrap();
+            }),
+        )
+        .map_err(|e| {
+            Error::Engine(format!("failed to find region {} err {:?}", region_id, e).into())
+        })?;
+
+    let result = rx2.recv().map_err(|e| {
+        Error::Engine(
+            format!(
+                "failed to receive from newly created channel {:?} for region {}",
+                e, region_id
+            )
+            .into(),
+        )
+    })?;
+    match result {
+        Some(local_region_info) => {
+            let local_region_epoch = local_region_info.region.region_epoch.unwrap();
+
+            // TODO(lance6717): we should only need to check conf_ver because we require all
+            // peers have SST on the disk, and does not care about which one is
+            // leader. But since check_sst_for_ingestion also checks epoch version,
+            // we just keep it here for now.
```

Comment:
I think it must check version too, because region split does not change conf_version.

Reply:
The function is_epoch_stale also checked epoch.version.